README.md

```diff
@@ -19,7 +19,7 @@ Technical resources for those working with or contributing to the project are av
 ## Development Environment
 
 - [Node.js](https://nodejs.org/) (v22 or higher)
-- [Docker](https://www.docker.com/) (v4.32.0 or higher)
```

Comment:
Is this newer version a requirement? recommended? or merely the latest?

Reply:
second thought, seems like something @MitchellShiell  changed.
I'll follow up with him separately...